.gitignore

```diff
@@ -1,2 +1,7 @@
 /target
 /Cargo.lock
+
+#Intellij Rust IDE
+.idea
+*.iml
+*.ipr
```

Comment:
don't think these should be here. here is some more information on how to configure git https://gist.github.com/subfuzion/db7f57fff2fb6998a16c locally to ignore those files.